scope.js

```diff
@@ -19,7 +17,7 @@ function outerFunction() {
 
   function innerFunction() {
     var world = 'World';
-    return "Hello World";
```

Comment:
We want to return the string 'Hello World' using the variables declared in the inner and outer functions rather then just returning the string 'Hello World' itself.